paper/paper.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'secuTrialR: Seamless interaction with clinical trial databases in R'
+tags:
+  - R software
+  - clinical trials
+  - data management
+  - descriptive statistics
+  - secuTrial
+authors:
+ - name: Patrick R. Wright
+   orcid: 0000-0002-1153-0846 
+   affiliation: "1, 3"
+ - name: Alan G. Haynes
+   orcid: 0000-0003-1374-081X
+   affiliation: "2, 3"
+ - name: Milica Markovic
+   orcid: 0000-0002-6973-6679
+   affiliation: "1, 3"
+affiliations:
+ - name: University Hospital Basel, Clinical Trial Unit, Basel, Switzerland
+   index: 1
+ - name: University of Bern, Clinical Trial Unit, Bern, Switzerland
+   index: 2
+ - name: Swiss Clinical Trial Organisation, Bern, Switzerland
+   index: 3
+date: 08 April 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+Elementary clinical trials have been conducted for hundreds of years [@meinert1986clinical]. The most famous early example
+is the proof that sailors scurvy can be cured by the consumption of citrus fruit [@lind_2014] performed by James Lind
+in the 18th century. Since these initial days of clinical research, trials have significantly evolved methodically, ethically
+and technologically. While it was viable and legitimate to collect clinical trials data in unversioned
+spread sheets in the past, this is no longer true and digital clinical data management systems (CDMS) have taken over.
+CDMS allow constraint based and version controlled data entry into a clinical trial database which ensures traceability, 
+integrity and quality of study data.  
+
+There is a vast market of heterogeneous CDMS solutions each with individual advantages and limitations [@kuchinke_etal_2010].
+One limitation can be the interaction with the data after it has been collected. Specifically, a CMDS may be
+tailored for optimal data capture while, at least to some extent, disregarding ease-of-use of study data after
+the conclusion of data entry. It is, however, vital that the interaction between data sources and data analysts is
+fast and seamless in order to avoid loss of valuable time due to technical overhead. This point has been prominently
+highlighted by the currently ongoing coronavirus pandemic [@callaway_etal_2020] in which issues have been reported
+regarding the timely transfer of information for the preparation of up-to-date infection
+counts [@spiegel_meldeluecke]. These issues led to confusion and may have ultimately
+delayed important actions. While this is a stark example it still serves to show how severe the influence
```

Comment:
is a comma missing after example?

Reply:
I don't know but I'll add it because my comma skills are lacking. So if you have any other comma suggestions I'm happy to hear them.